src/riak_core_util.erl

```diff
@@ -150,6 +155,55 @@ replace_file(FN, Data) ->
             Err
     end.
 
+%% @doc Similar to {@link file:consult/1} but will consult files that
+%% are not valid unicode. Specifially files written with
+%% `io_lib:format("~p,", io_list())' in r16 before `utf8' became the
+%% default encoding.
+-spec consult(file:filename()) -> {ok, [term()]} | {error, Err :: term()}.
+consult(FName) ->
+    case file:consult(FName) of
+        {ok, Terms} ->
+            {ok, Terms};
+        {error, _Reason} ->
+            latin_consult(FName)
```

Comment:
Yeah. You know what, I really don't like code that may or may not do X in future. I think I will just move this all to riak_kv_vnode_status_mgr to handle that one specific problem.